pkgs/applications/editors/texmaker/default.nix

```diff
@@ -9,7 +9,7 @@ mkDerivation rec {
     sha256 = "0qczc2r01vhap11xmqizwbq21ggn4yjrxim8iqjxaq9w1rg2x9dz";
   };
 
-  buildInputs = [ qtbase qtscript poppler zlib qtwebengine ];
+  buildInputs = [ qtbase qtscript poppler zlib qtwebengine wrapGAppsHook ];
   nativeBuildInputs = [ pkg-config poppler qmake ];
```

Comment:
```suggestion
  buildInputs = [ qtbase qtscript poppler zlib qtwebengine ];
  nativeBuildInputs = [ pkg-config poppler qmake wrapGAppsHook ];
```

Reply:
Opps thanks for spotting :)

Updated in latest push https://github.com/NixOS/nixpkgs/pull/227005/files#diff-6d83d8c41bdb1c85cf61904fbcc3d39b9f4fc7d1e46a9f7143b800c9a626a737R13